Dockerfile.basestation

```diff
@@ -32,14 +30,16 @@ RUN udevadm trigger
 
 COPY src .
 
-RUN . /opt/ros/$ROS_DISTRO/install/setup.sh && \
+# build all packages defined in basestation_launch file
+RUN export PYTHONWARNINGS=ignore:::setuptools.command.install,ignore:::setuptools.command.easy_install,ignore:::pkg_resources && \
```

Comment:
Ah yes, the ignore statement

Reply:
colcon uses a deprecated function to install our python packages
three possible remedies:
fix colcon
change to c++
or ignore python's complaining

I chose to gag python